service/src/main/java/bio/terra/landingzone/library/configuration/stairway/LandingZoneStairwayConfiguration.java

```diff
@@ -6,20 +6,17 @@
 import bio.terra.common.stairway.StairwayProperties;
 import bio.terra.landingzone.library.configuration.LandingZoneAzureConfiguration;
 import bio.terra.landingzone.library.configuration.LandingZoneStairwayProperties;
-import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class LandingZoneStairwayConfiguration {
   @Bean
-  @ConfigurationProperties(prefix = "landingzone.stairway")
```

Comment:
Just tidying up a little.

The configuration property selector only needs to be included once. Most of the existing ones had it on the class itself, so I kept it there for consistency.